src/rest/rest.ts

```diff
@@ -0,0 +1,46 @@
+import { checkRateLimits } from "./check_rate_limits.ts";
+import { cleanupQueues } from "./cleanup_queue.ts";
+import { createRequestBody } from "./create_request_body.ts";
+import { handlePayload } from "./handle_payload.ts";
+import { processQueue } from "./process_queue.ts";
+import { processRateLimitedPaths } from "./process_rate_limited_paths.ts";
+import { processRequest } from "./process_request.ts";
+import { processRequestHeaders } from "./process_request_headers.ts";
+import { RequestManager } from "./request_manager.ts";
+import { runMethod } from "./run_method.ts";
+
+export const rest: RestCache = {
```

Comment:
Shouldn't this be `let` so users can redeclare the variable?

Reply:
Nope, u can edit a const {} without redeclaring

Reply:
yes and no:
```ts
const foo = {
  bar: 20,
}

// works
foo.bar = 40;

// Cannot assign to 'foo' because it is a constant.
foo = {
  bar: 40,
}
```